internal/pkg/cli/task_run.go

```diff
@@ -437,22 +437,21 @@ func BuildTaskRunCmd() *cobra.Command {
 	}
 	cmd := &cobra.Command{
 		Use:   "run",
-		Short: "Run a one-off task",
-		Long:  `Run a one-off task with configurations such as cpu-units, memory, image, etc.`,
+		Short: "Run a one-off task on Amazon ECS.",
 		Example: `
-Run a task with default setting. You will be prompted to specify a task group name and an environment for the tasks to run in.
+Run a task with default settings. 
+You will be prompted to specify a task group name and an environment for the tasks to run in.
 /code $ copilot task run
 Run a task named "db-migrate" in the "test" environment under the current workspace.
 /code $ copilot task run -n db-migrate --env test
-Starts 4 tasks with 2GB memory, run a particular image, and run with a particular task role.
-/code $ copilot task run --num 4 --memory 2048 --image=python --task-role migrate-exec-role
+Run 4 tasks with 2GB memory, an existing image, and a custom task role.
+/code $ copilot task run --num 4 --memory 2048 --image=rds-migrate --task-role migrate-role
```

Comment:
It says [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions.html#cfn-ecs-taskdefinition-containerdefinition-image) "Images in official repositories on Docker Hub use a single name (for example, `ubuntu` or `mongo`)." Should we provide an actual office image in the example here?

Reply:
I just wanted to give an image name that would hint at what kindof stuff they could do with `task run` that's why I put `rds-migrate` there